### PR TITLE
Changes to models testing functions to bring them in sync with schemas

### PR DESCRIPTION
**Pull request by nden**
The changes in schemas include
- photometry changes - spacetelescope/rad#114
- making photometry an optional property - spacetelescope/rad#115
- adding `p_keywords` - spacetelescope/rad#105
- adding `level0_compression` attribute - spacetelescope/rad#104
- adding `gw_fgs_mode` to `guide_star` - spacetelescope/rad#103

This PR also updates versioons of some dependencies in setup.cfg and points to rad/main

### CODE REVIEW

File: src/roman_datamodels/testing/utils.py [hunk 1]
```diff
@@ -397,7 +399,7 @@ def mk_basic_meta():
     return meta
 
 
-def mk_common_meta():
+def mk_common_meta(photometry=True):
```
Comment (ddavis-stsci) [inline src/roman_datamodels/testing/utils.py:402]: Should this be photmetry=False as the default if we are removing it as a required element? and set it to true once we want it?

**Review by ddavis-stsci — APPROVED** [review]
Besides my comment below (which is really about style) LGTM.

**Review by cshanahan1 — APPROVED** [review]

File: src/roman_datamodels/testing/utils.py (hunk 1 above)
Comment (nden) [reply src/roman_datamodels/testing/utils.py]: removed from `common` now

**Comment by nden** [comment]
This PR includes the commits from #65.
It also points to the rad/main until a rad release is made.

**Pull request merged**